src/components/WalletConnect/ModalButton.js

```diff
@@ -11,20 +11,20 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     marginLeft: 12,
     borderWidth: 2,
-    borderColor: '#cecece',
+    borderColor: COLORS.borderColor,
```

Comment:
Change from `#cecece` to `#eeeeee`

Reply:
Why?

Reply:
As discussed, this is close enough to the current border color to compose the app theme.

Reply:
Reverted to original color on [2e29c07](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/2e29c070354d1fae1ee605c2884a2c6ff80cb0b2)